public/app/plugins/panel/graph/module.ts

```diff
@@ -232,6 +232,7 @@ class GraphCtrl extends MetricsPanelCtrl {
 
   changeSeriesColor(series, color) {
     series.color = color;
+    series.bars.fillColor = color;
```

Comment:
this feels wrong, maybe better to move this to a function in the time series class?